instances/administrationType/intraperitoneal.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/administrationType/intraperitoneal",
```

Comment:
I would be more specific and say `intraperitonealInjection` and change respectively the definition and name (which is anyway not correct; copy / past error I assume)

Reply:
definition suggestion: "An 'intraperitoneal injection' is the administration of a substance into the peritoneum (body cavity) via a needle or tube."